internal/core/autocomplete.go

```diff
@@ -28,8 +37,20 @@ type AutoCompleteNode struct {
 	Children map[string]*AutoCompleteNode
 	Command  *Command
 	ArgSpec  *ArgSpec
-	IsArg    bool
-	// TODO: support flags in the autocomplete node
+	Type     AutoCompleteNodeType
+}
+
+func (n *AutoCompleteNode) addGlobalFlags() {
+	n.Children["--access-key"] = NewFlagAutoCompleteNode(n, nil)
```

Comment:
can we add this at the same level as flag declaration?

Reply:
Internal ticket: APIGW-1758